src/util/unwind.hpp

```diff
@@ -14,28 +9,31 @@ namespace mc {
 namespace util {
 
 #ifdef WITH_UNWIND
```

Comment:
Given that the only reason `libunwind.h` needs to be included here is to get the definition of `unw_word_t`, which itself is an integral type, could we keep the #ifdefs and such inside `unwind.cpp`, hidden from the rest of the code, and use e.g.
```
struct source_location {
    std::string name;
    std::uintptr_t address;
};
```
which does not depend upon the architecture-specific int size used for `unw_word_t`?

This has the nice side effect of keeping the global namespace free of the libunwind-specific enums and typedefs.